content/articles/How-to-start-up-with-Socket-Programming-with-Java/index.md

```diff
@@ -180,19 +181,19 @@ Client: BYE
 Closing connection
 ```
 
-### Client side Programming
-# Establish a Socket Connection
-This is an instance of a socket class and can be obtained in two ways: 
+### Client-side Programming
+#### Establish a Socket Connection
+A Socket instance can be obtained in two ways: 
     
-1. On the server side it can be used as a return value of the accept ( ) method.
+1. On the server side it can be used as a return value of the accept method.
 2. The builder of socket objects accepts the hostname of the server on which the server operates and the server's port number.
 
 ```java
 socket = new Socket("localhost", 5000);
 ```
 
 localhost - is a domain name like google.com, localhost takes you to your computer. It is resolved to IP address 127.0.0.1 .
-port - Port – This is built to connect with the server via the port number socket object.
+The second argument is a port number.
 Two computers communicate through sockets.
 
 Client and Server communicate by writing to an output stream and reading from an input stream on the socket.
```

Comment:
>Client and Server communicate by writing to an output stream and reading from an input stream on the socket.

You already mentioned this in the other part about the server-side application. Instead, I would transition into the next part by saying:

>Like before, we create I/O streams via the socket object: